src/data/faq.json

```diff
@@ -159,6 +159,16 @@
                             "<ul><li>'NEXT STEPS' -> 'Enter TAN'</li></ul>"
                         ]
                     },
+                    {
+                        "title": "Red Card? How to test",

```

Comment:
"How to test" sounds like it is about testing the Red Card, not the person. Also the FAQ article more about sharing the result than about testing.

Suggest instead:
"Red Card? How to get tested and best use the app"